README.md

```diff
@@ -7,6 +7,10 @@
 
 Zigbee Home Automation integration with zigpy allows you to connect one of many off-the-shelf Zigbee adapters using one of the available Zigbee radio library modules compatible with zigpy to control Zigbee based devices. There is currently support for controlling Zigbee device types such as binary sensors (e.g., motion and door sensors), sensors (e.g., temperature sensors), lightbulbs, switches, and fans.
 
+This software is aimed at application developers who wish to incorporate or integrate Zigbee functionality into their applications. The ultimate goal is for zigpy to be a free software library that interfaces with Zigbee controllers (that are called "coordinators" in the Zigbee specification), allowing anyone who integrates zigpy to create applications to control devices on a Zigbee network, without requiring very depth knowledge of the Zigbee protocols. The zigpy projects is not about creating Zigbee devices (routes and end devices). The project consists of the main zigpy library, wrappers for Zigbee radios from different manufacturers, and supporting projects, all written in Python.
```

Comment:
Thanks! Some spelling/grammar fixes:

 1. `very depth` -> `deep` or `in-depth`
 2. *The zigpy projects is not about creating Zigbee devices (routes and end devices).*: I think this is covered by the second sentence that explains that zigpy interfaces to coordinators.

Reply:
Updated now however I did not remove that sentence about zigpy not being about Zigbee devices because currently, the readme does not explain Zigbee terminology in regards to differences between a "coordinator", "router" and "end device" so think it does not hurt to slip in a short clarification there about what zigpy is not meant to be for.